package.json

```diff
@@ -45,6 +45,7 @@
     "express-handlebars": "^5.1.0",
     "express-session": "^1.17.1",
     "helmet": "^4.1.1",
+    "materialize-css": "^1.0.0-rc.2",
```

Comment:
not sure why we are calling to a materialize-css package? might not need this?

Reply:
Right. We don't need it. How do I remove it?

Reply:
npm uninstall materialize-css